src/wallet/testdata/duplicate_wallets/test3.1.wlt

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
Can't test via NewService or LoadWallets then, because it will load the other fault wallets as well and then it wont come to the point, where the duplicate wallets are being loaded and the actual test is being executed.